tools/fasta_compute_length/fasta_compute_length.xml

```diff
@@ -1,9 +1,13 @@
 <tool id="fasta_compute_length" name="Compute sequence length" version="1.0.0">
 	<description></description>
-	<command interpreter="python">fasta_compute_length.py $input $output $keep_first</command>
+	<command interpreter="python">fasta_compute_length.py $input $output $keep_first $keep_first_word</command>
 	<inputs>
 		<param name="input" type="data" format="fasta" label="Compute length for these sequences"/>
 		<param name="keep_first" type="integer" value="0" label="How many title characters to keep?" help="'0' = keep the whole thing"/>
+        <param name="keep_first_word" type="boolean" truevalue="id_only" falsevalue="id_and_desc"
```

Comment:
It seems this file is mixing tabs with spaces, so here the indentation looks wrong.